core_backend/app/question_answer/models.py

```diff
@@ -8,13 +8,14 @@
 """
 
 from datetime import datetime, timezone
-from typing import List
+from typing import Dict, List
```

Comment:
can we start using native types instead of importing from `typing`?

Reply:
TIL that importing these from typing is [deprecated](https://peps.python.org/pep-0585/). Sounds good!